modules/cluster-logging-deploy-clo.adoc

```diff
@@ -5,66 +5,36 @@
 [id="cluster-logging-deploy-clo_{context}"]
 = Install the Cluster Logging Operator using the web console
 
-You can use the {product-title} web console to install the Cluster Logging Operator.  
+You can use the {product-title} web console to install the Cluster Logging Operator.
 
 [NOTE]
```

Comment:
This note should no longer be necessary since the namespace is created automatically when you install the operator.